docs/fastdds/use_cases/reduce_memory/reduce_memory.rst

```diff
@@ -0,0 +1,97 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+
+.. _use-case-reduceMemory:
+
+Reduce the usage of memory
+==========================
+
+A great number of modern systems have tight constraints over available memory. Reducing the memory usage is critical for
+these applications. Reducing memory consumption of a Fast-DDS application can be achieved through a different number of
+approaches, mainly through architectural restructuring, but also via the limiting of resources and by avoiding static
+allocations.
+
+If a system has a low memory limit it is recommended to limit the maximum resources of the application to ensure that it
+will run properly. To lower the usage during runtime, allocations can be set dynamically to ensure the lowest amount of
+memory is always in use.
+
+QoS Adjustments
+---------------
+
+ResourceLimitsQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^^
+Limit the resources to the application's needs. The lowest it can be reduced to is one sample per |DataWriter-api| and
```

Comment:
I think we should explain here what this QoS does, and link to its sections in the docs. I think it's important to state what parameters are to be modified and why